yarn.lock

```diff
@@ -1335,6 +1335,11 @@
   resolved "https://registry.yarnpkg.com/@types/events/-/events-3.0.0.tgz#2862f3f58a9a7f7c3e78d79f130dd4d71c25c2a7"
   integrity sha512-EaObqwIvayI5a8dCzhFrjKzVwKLxjoG9T6Ppd5CEo07LRKfQ8Yokw54r5+Wq7FaBQ+yXRvQAYPrHwya1/UFt9g==
 
+"@types/expect@1.20.4":
```

Comment:
I suspect we can drop `expect` wholesale in lieu of `assert`.

@avatarneil you're with me on team `assert`, right?